mvp-spec/partials-updater/src/lexer.ts

```diff
@@ -35,7 +37,7 @@ export function lex(content: string): LexerToken[] {
                 text: line
             };
             tokens.push(token);
-        } else if (line == PartialEndKey) {
+        } else if (line.match(PartialEnd)) {
```

Comment:
What is the rational of changing this?

Reply:
It requires a regex to handle multiple types of linebreaks.
Before it was testing against a hardcoded linebreak.